src/components/application_manager/src/commands/mobile/system_request.cc

```diff
@@ -299,8 +289,8 @@ class QueryAppsDataValidatorTemplate {
               language[*iter][json::ttsName];
           // ttsName is string
           if (smart_objects::SmartType_String == ttsNameObject.getType()) {
-            utils::custom_string::CustomString ttsName =
-                language[*iter][json::ttsName].asCustomString();
+            std::string ttsName =
```

Comment:
Fixed all std::string variables to be const. [81c7cf2d0e1](https://github.com/smartdevicelink/sdl_core/pull/433/commits/81c7cf2d0e1ee43ef5fa0f443165853d1bbe6484)